CHANGELOG.md

```diff
@@ -282,7 +282,7 @@
 - Added support for `break` and `skip` values in `walk()` to control traversal
 - Added `List#reduce()` and `List#reduceRight()` methods
 - Bumped `mdn-data` to 2.0.12
-- Exposed version of the lib (i.e. `import { version } from 'css-tree'`)
+- Exposed version of the lib (i.e. `import { version } from '@eslint/css-tree'`)
```

Comment:
Is this change intended? There are more old entries in the changelog where the imported package is still named `css-tree`.

Reply:
Not intended. I think this just got caught up in a global find-and-replace (not sure why the others didn't).